src/utils.ts

```diff
@@ -715,13 +715,7 @@ export const getResponseSchemaValidator = (
 
 	const compile = (schema: TSchema, references?: TSchema[]) => {
 		const cleaner = (value: unknown) => {
-			if (!value || typeof value !== 'object')
-				return Value.Clean(schema, value)
-
-			if (Array.isArray(value)) value = Value.Clean(schema, value)
-			else value = Value.Clean(schema, value)
-
-			return value
+			return Value.Clean(schema, structuredClone(value))
```

Comment:
@SaltyAom correct me if I'm wrong, but it seems to me all 3 logical branches end up doing the same thing?